extras/test/ConditionTests/on_less_tests.cpp

```diff
@@ -146,6 +146,30 @@ TEST(ConditionTests, handleActionTestsForDouble) {
   // nothing should happen
   channel->setNewValue(-1, -1, -1, -1, 25);
   cond->handleAction(Supla::ON_CHANGE, action1);
+  
+  // RGB use int type on channel value
+  channel->setType(SUPLA_CHANNELTYPE_RGBLEDCONTROLLER);
+
+  channel->setNewValue(-1, -1, -1, -0, -1);
+  // channel should be initialized to 0, so condition should be met
+  cond->handleAction(Supla::ON_CHANGE, action1);
+
+  channel->setNewValue(-1, -1, -1, 100, -1);
+
+  // 100 is not less than 15, so nothing should happen
+  cond->handleAction(Supla::ON_CHANGE, action2);
+
+  // Values below 0 should be ignored
+  channel->setNewValue(-1, -1, -1, -1, -1);
```

Comment:
In case of RGBW, -1 here doesn't mean "value below 0". It actually means that particular color/brightness is not modified.
You can keep check with all values set to -1, but please update comment to i.e. "RGBW values -1 doesn't change actual value on channel, so it won't trigger action"

Reply:
ok